DistFiles/localization/en/Bloom.xlf

```diff
@@ -4231,7 +4227,6 @@
         <note>ID: PublishTab.PdfNotSavedWhy</note>
         <note>explanation that file was not saved displayed in a message box</note>
       </trans-unit>
-      <!-- for 5.4
       <trans-unit id="PublishTab.PdfPrint.Button">
```

Comment:
I think this section actually should stay commented out (and I guess the comment changed to 5.5).
We didn't ship this yet.
Is that right, @gmartin7 ?